elibrary-hibernate/pom.xml

```diff
@@ -120,7 +120,7 @@
 		<dependency>
 			<groupId>org.hibernate</groupId>
 			<artifactId>hibernate-core</artifactId>
-			<version>${org.hibernate.version}</version>
+			<version>5.4.10.Final</version>
```

Comment:
We were using parametrized version (in brackets), it should be done in the same this time too.